unskript-ctl/unskript_audit_cleanup.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2023 unSkript.com
+# All rights reserved.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE
+#
+#
+
+import os
+import yaml
+import ZODB
+import datetime
+import ZODB.FileStorage
+from ZODB import DB
+
+# LIST OF CONSTANTS USED IN THIS FILE
+
+PSS_DB_PATH="/unskript/db/unskript_pss.db"
+EXECUTION_DIRECTORY="/unskript/data/execution"
```

Comment:
why are we duplicating this definitions everywhere? why not use utils.py?

Reply:
Moved all these constants to unskript_utils.py